templates/template.yaml

```diff
@@ -1,6 +1,6 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Transform: AWS::Serverless-2016-10-31
-Description: Amazon S3 Find and Forget (uksb-1q2j8beb0) (version:v0.45)
+Description: Amazon S3 Find and Forget (uksb-1q2j8beb0) (version:)
```

Comment:
there must be something interesting happening here as this should stay 0.45 if you are not bumping the version. Are your pre-commit scripts running correctly?

Reply:
they aren't. 😀